package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "description": "Streamlined DOM Events for most.js",
   "main": "dist/dom-event.js",
+  "typings": "type-definitions/dom-event.d.ts",
```

Comment:
I don't have any fact or reasoning to back this, but I feel like 2 is the better option